azure/template.json

```diff
@@ -51,37 +58,31 @@
         "internalApiCertificateName": {
             "type": "string"
         },
-        "SqlConnectionstring":{
-            "type": "string"
-        },
-        "AllowedHosts":{
+        "workerAspSize": {
             "type": "string"
```

Comment:
Default value to be 1?

"defaultValue": "1"

Reply:
Added